Sources/Penny/SOChecker.swift

```diff
@@ -0,0 +1,147 @@
+import Logging
+import DiscordBM
+import Markdown
+import Foundation
+
+actor SOChecker {
+
+    struct Storage: Sendable, Codable {
+        var lastCheckDate: Date? = nil
```

Comment:
```suggestion
        var lastCheckDate: Date?
```
Explicit `nil` initialization of optionals is redundant.

Reply:
eh, ok